website/source/guides/operations/upgrade/upgrade-specific.html.md

```diff
@@ -15,6 +15,84 @@ details provided for their upgrades as a result of new features or changed
 behavior. This page is used to document those details separately from the
 standard upgrade flow.
 
+## Nomad 0.9.0
+
+### Task Driver Plugins
+
+All task drivers have become plugins in Nomad 0.9.0. There are no user visible
+differences between 0.8 and 0.9 drivers except for LXC. There is a new method
+for client driver configuration options, but existing `client.options` settings
+are supported in 0.9. See [plugin configuration][plugin-stanza] for details.
+
+#### LXC
+
+LXC is now an external plugin and must be installed separately. See [the LXC
+driver's documentation][lxc] for details.
+
+### Structured Logging
+
+Nomad 0.9.0 switches to structured logging. Any log processing on the pre-0.9
+log output will need to be updated to match the structured output.
+
+Structured log lines have the format:
+
+```
+# <Timestamp> [<Level>] <Component>: <Message>: <KeyN>=<ValueN> ...
+
+2019-01-29T05:52:09.221Z [INFO ] client.plugin: starting plugin manager: plugin-type=device
+```
+
+Values containing whitespace will be quoted:
+
+```
+... starting plugin: task=redis args="[/opt/gopath/bin/nomad logmon]"
+```
+
+### HCL2 Transition
+
+Nomad 0.9.0 begins a transition to [HCL2][hcl2], the next version of the
+HashiCorp configuration language. While Nomad has begun integrating HCL2,
+users will need to continue to use HCL1 in Nomad 0.9.0 as the transition is
+incomplete.
+
+The only backward incompatible change is that `task.config` stanzas now only
+support dotted nested object notation in interpolation. Interpolating variables
+that have multiple consecutive dots in the `task.config` stanza requires using
+a new `env["..."]` map:
+
+```hcl
+env {
+  # Note the multiple consecutive dots
+  image...version = "3.2"
+
+  # Valid in both v0.8 and v0.9
+  image.version = "3.2"
+}
+
+# v0.8 task config stanza:
+task {
+  driver = "docker"
+  config {
+    image = "redis:${image...version}"
+  }
+}
+
+# v0.9 task config stanza:
+task {
+  driver = "docker"
+  config {
+    image = "redis:${env["image...version"]}"
+  }
+}
+```
+
+This only affects users who interpolate unusual variables with multiple
+consecutive dots in their task `config` stanza. All other interpolation is
+unchanged.
+
+Since HCL2 uses dotted object notation for interpolation users should
+transition away from variable names with multiple consecutive dots.
+
```

Comment:
I think we should mention preemption here as a headsup to operators. e,g "Nomad 0.9 adds preemption support for system jobs. If a system job is submitted that has a higher priority than other running jobs on the node, and the node does not have capacity remaining, Nomad may preempt those lower priority allocations to place the system job. See [preemption][link] for more details"

Reply:
Oh wow, absolutely! Good catch. Putting that one first.